building/javascript-sdk.md

```diff
@@ -56,17 +50,19 @@ import { connect } from "@tableland/sdk";
 const tbl = await connect({ network: "testnet" });
 
 const createRes = await tbl.create(
-  `CREATE TABLE mytable (name text, id int, primary key (id));`
+  4,  // Chain Id, note that since we connected to rinkeby testnet above this **MUST** be 4
+  `name text, id int, primary key (id);`, // table schema definition
+  `mytable` // optional perfix that will be added to "queryable name"
 );
 
-// `queryableName` will be the table name you chose with the
-// table id separated by and underscore 
+// `queryableName` will be the prefix, if you chose one, with the chain id and
+// table id separated by underscores 
 const queryableName = createRes.name;
-console.log(queryableName); // e.g. mytable_1
+console.log(queryableName); // e.g. mytable_4_1
```

Comment:
I can't do a "suggestion" on delete lines for some reason, so here's the suggestion:

```
// `name` will be the prefix, if supplied, with the chain id and
// table id separated by underscores.
console.log(createRes.name)
```

Now that name is much simpler to reason about, I don't think we need to have a separate variable for it here. That makes the example simpler, and highlights that they'll get the name from the response, and they should just use that.

Reply:
The above obviously implies that we'll need to change all use of queryableName => createRes.name. The alternative is to simply destructure the create response into name and whatever else we want/need in the example for a cleaner demo?